eval.go

```diff
@@ -26,3 +26,15 @@ func evalCond(cond string, store map[string]interface{}) (bool, error) {
 	}
 	return tf.(bool), nil
 }
+
+func evalCount(count string, store map[string]interface{}) (int, error) {
+	r, err := expr.Eval(count, store)
+	if err != nil {
+		return 0, err
+	}
+	c, ok := r.(int)
+	if !ok {
+		return 0, fmt.Errorf("invalid count: %v", count)
+	}
```

Comment:
[IMO] Since the result of `expr.Eval` is of multiple types, it would be better to convert it to int appropriately.

```suggestion
	var c int
	switch v := r.(type) {
	case string:
		c, _ = strconv.Atoi(v)
	case int64:
		c = int(v)
	case float64:
		c = int(v)
	case int:
		c = v
	default:
		return 0, fmt.Errorf("invalid count: evaluated %s, but got %T(%v)", count, r, r)
	}
```

Reply:
Thank you!!